src/lib/annotations/doc/README.md

```diff
@@ -0,0 +1,104 @@
+# Document and Presentation Annotations
+
+The document and presentation viewers supports highlight comment, highlight only, draw and point annotations.
+
+## Screenshot
+
+![Screenshot of document point annotations](../../../../images/doc_point.png)
```

Comment:
These aren't rendering for some reason

Reply:
Whoops forgot to add them... I'll add that part afterwards